themes/cardinal_service/lib/components/opportunity-row/opportunity_row.html.twig

```diff
@@ -11,38 +11,49 @@
       </h3>
 
       <div class="su-opportunity-body-content">
-        {% if application_deadline|render %}
           <div class="su-opportunity-row-application-deadline">
             <div>
               <h4>{{ 'Application Deadline'|t }}:</h4>
+              {% if application_deadline|render %}
               <div>{{ application_deadline }}</div>
+              {% else %}
+                <div><em>{{ 'N/A'|t }}</em></div>
+              {% endif %}
```

Comment:
I sneaked this logic in the template in to the `8.x-1.x` branch. I'm going to revert this file and merge in the base branch

Reply:
Sounds good 😄  Thanks Mike!